### PR TITLE
docs: how to prevent closing challenge upon back press

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #131

### CODE REVIEW

File: sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java [hunk 1]
```diff
@@ -283,6 +283,11 @@ private View prepareRootView(@NonNull LayoutInflater inflater,
                 return true;
             }
 
+            final boolean ignoreBackPress = Boolean.FALSE.equals(config.getStopVerificationOnBackPress());
+            if (ignoreBackPress) {
+                return true;
+            }
+
             return webViewHelper != null && webViewHelper.shouldRetry(
```
Comment (DSergiu) [inline sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java:291]: Can't `config.getStopVerificationOnBackPress()` already be achieved with a custom `config.retryPredicate`?

File: sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java [hunk 2]
```diff
@@ -283,6 +283,11 @@ private View prepareRootView(@NonNull LayoutInflater inflater,
...
                     new HCaptchaException(HCaptchaError.CHALLENGE_CLOSED));
```
Comment (DSergiu) [inline sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java:292]: Can't `config.getStopVerificationOnBackPress()` already be achieved with a custom `config.retryPredicate`?

Comment (CAMOBAP) [reply sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java:292]: I think no.

`config.retryPredicate` is about retrying verification when it failed for some reason. `config.stopVerificationOnBackPress=false` about ignoring user's back presses/gestures

Comment (CAMOBAP) [reply sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java]: Good point, PR updated accordingly

**Review by DSergiu — APPROVED** [review]

**Pull request merged**